PKGBUILD

```diff
@@ -22,13 +26,18 @@ build()
     -des -Dprefix=/usr \
     -Dcc="cc -D_GNU_SOURCE" \
     -Dvendorprefix=/usr \
-    -Dprivlib="/usr/lib/perl5/${pkgver}" \
-    -Darchlib="/usr/lib/perl5/${pkgver}/$(arch)-linux" \
-    -Dsitelib="/usr/lib/perl5/site_perl/${pkgver}" \
-    -Dvendorlib="/usr/lib/perl5/vendor_perl/${pkgver}" \
-    -Dvendorarch="/usr/lib/perl5/vendor_perl/${pkgver}/$(arch)-linux" \
+    -Dprivlib=/usr/share/perl5/core_perl \
+    -Darchlib=/usr/lib/perl5/$_baseversion/core_perl \
+    -Dsitelib=/usr/share/perl5/site_perl \
+    -Dsitearch=/usr/lib/perl5/$_baseversion/site_perl \
+    -Dvendorlib=/usr/share/perl5/vendor_perl \
+    -Dvendorarch=/usr/lib/perl5/$_baseversion/vendor_perl \
     -Dman1dir=/usr/share/man/man1 \
     -Dman3dir=/usr/share/man/man3 \
+    -Dscriptdir=/usr/bin/core_perl \
+    -Dsitescript=/usr/bin/site_perl \
+    -Dvendorscript=/usr/bin/vendor_perl \
```

Comment:
I'll prefer to keep the execs in the default path and drop perlbin.sh, which will cause perl to install its binaries (including perl interpreter/cpan) to `/usr/bin/core_perl`. This'll break chroot-based environment (they don't respect /etc/profile.d) and I don't see its value.